### PR TITLE
[Common] Enable snapshot version

### PR DESCRIPTION
**Pull request by kittinunf**
## Description

As our discussion, it looks like it is gonna be beneficiary to everyone if we do support this *officially*. I think we will publish the `-SNAPSHOT` everytime we hit `master` branch.

## Type of change

Check all that apply

- [x] Chores

## How Has This Been Tested?

In case you did not include tests describe why you and how you have verified the
changes, with instructions so we can reproduce. If you have added comprehensive
tests for your changes, you may omit this section.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, if necessary
- [ ] My changes generate no new compiler warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Inspect the bytecode viewer, including reasoning why

### CODE REVIEW

File: deploy_bintray.sh [hunk 1]
```diff
@@ -14,3 +14,18 @@ if [[ "$TRAVIS_BRANCH" == */release-v* ]]; then
...
+if [[ "$TRAVIS_BRANCH" == master ]]; then
+
+  echo "We're on master branch, deploying SNAPSHOT"
+
+  for i in $(ls -d */);
+  do
+    m=${i%%/}
+    if [[ $m == fuel* ]]; then
+      echo ">> Deploying $m ..."
+      ./gradlew :$m:clean :$m:build :$m:bintrayUpload -PBINTRAY_USER=$BINTRAY_USER -PBINTRAY_KEY=$BINTRAY_KEY -PdryRun=false -Ppublish=true
```
Comment (iNoles) [inline deploy_bintray.sh]: I think we should exclude test and sample here

Comment (kittinunf) [reply deploy_bintray.sh]: I think it should be okay because test and sample module don't have `bintrayUpload` configured.

**Comment by kittinunf** [comment]
Shall we merge this?

**Comment by SleeplessByte** [comment]
Don't we have that issue with 10GB limit etc? Or what were the concerns exactly?

**Comment by kittinunf** [comment]
I am not sure though. But I can write a script to prune them if we hit the space limit.

**Comment by kittinunf** [comment]
It looks like we couldn't find a good way for this, maybe I will close it for now.

**Comment by NikkyAI** [comment]
https://jitpack.io/#kittinunf/Fuel is still working just fine
just use `group = "com.github.kittinunf.Fuel", name = subproject, version = "-SNAPSHOT"`
subprojects owuld be `fuel`, `fuel-coroutines`, etc

**Pull request closed without merging**